packages/channels/botpress-channel-rocketchat/src/views/index.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
```

Comment:
For channels we don't show configuration screens anymore (i.e. only JSON config)